### PR TITLE
Rename firefish-js to goblin-js

### PR DESCRIPTION
**Pull request by analytik**
name: Pull Request
about: Create a pull request
title: "[PR]: "
body:
  - type: markdown
    attributes:
      value: |
                Thanks for taking the time to make Firefish better! It's not required, but please consider using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) when making your commits. If you use VSCode, please use the [Conventional Commits extension](https://marketplace.visualstudio.com/items?itemName=vivaxy.vscode-conventional-commits).
  - type: textarea
    id: about
    attributes:
      label: What does this PR do?
      description: Please give us a brief description of what this PR does.
      placeholder: Makes Firefish so amazing by...
    validations:
      required: true
  - type: checkboxes
    id: terms
    attributes:
      label: Contribution Guidelines
      description: By submitting this issue, you agree to follow our [Contribution Guidelines](https://git.joinfirefish.org/firefish/firefish/src/branch/develop/CONTRIBUTING.md)
      options:
        - label: I agree to follow this project's Contribution Guidelines
          required: true
        - label: I have made sure to test this pull request
          required: true
        - label: I have made sure to run `pnpm run format` before submitting this pull request
          required: true

lol ^

### CODE REVIEW

**Comment by analytik** [comment]
Anyway I have not actually tested these changes, so this is more just to test the PR workflow (it's been like 8 years since i last made a PR on GitHub, as embarrassing as it might sound). Will try to test tomorrow if i have time, and then work on the documentation / k8s manifests / bare metal installation guide and scripts.

File: Dockerfile [hunk 1]
```diff
@@ -1,73 +1,131 @@
...
-COPY packages/backend/native-utils packages/backend/native-utils/
+COPY --chown=goblin:goblin packages/backend/native-utils packages/backend/native-utils/
 
 # Compile native-utils
 RUN pnpm run --filter native-utils build
 
 # Copy in the rest of the files to compile
-COPY . ./
+COPY --chown=goblin:goblin . ./
 RUN env NODE_ENV=production sh -c "pnpm run --filter '!native-utils' build && pnpm run gulp"
```
Comment (analytik) [inline Dockerfile:73]: also, offtopic but like. gulp. so many war flashbacks. super low priority obviously but I'd be glad if this old man could get sent ~behind barn~ to greener pastures

**Comment by ruler501** [comment]
I'm not sure what all is needed but section 5a of the AGPL requires prominent information on what this was forked from and when.

**Comment by analytik** [comment]
Will start over and this time not try to change 9999 files :D

**Pull request closed without merging**